packages/mobile/src/redux/migrations.test.ts

```diff
@@ -68,4 +68,16 @@ describe('Redux persist migrations', () => {
     expect(migratedSchema.identity.startedVerification).toBeUndefined()
     expect(migratedSchema.identity.hasSeenVerificationNux).toBeTruthy()
   })
+
+  it('work for v4 to v5', () => {
```

Comment:
I don't believe v4 has shipped to users yet so no need for a v5. But it's fine either way.
In fact, we should blow these away and start fresh before the next deployment because we can only do that when the update coincides with a network reset.

Reply:
Ok yes I'll leave them and we can reset all of these in another PR.